api/src/pcapi/core/search/__init__.py

```diff
@@ -432,6 +438,17 @@ def unindex_all_offers() -> None:
         logger.exception("Could not unindex all offers")
 
 
+def reindex_venues_from_offers(offer_ids: Iterable[int]) -> None:
+    """
+    Get the offers' venue ids and reindex them
+    """
+    query = Venue.query.join(Offer).filter(Offer.id.in_(offer_ids)).with_entities(Venue.id).distinct()
```

Comment:
(perf) Cette ligne fait une jointure entre les tables `offer` et `venue`. On devrait pouvoir récupérer les identifiants de lieux sans jointure avec quelque chose comme ça : 

```suggestion
    query = Offer.query.filter(Offer.id.in_(offer_ids)).with_entities(Offer.venueId).distinct()
```